python/taichi/lang/transformer.py

```diff
@@ -877,9 +877,49 @@ def visit_Return(self, node):
 class ASTTransformerChecks(ASTTransformerBase):
     def __init__(self, func):
         super().__init__(func)
+        self.has_return = False
+        self.in_static_if = False
 
     def visit_Call(self, node):
         if isinstance(node.func, ast.Name):
             node.args = [node.func] + node.args
             node.func = self.parse_expr('ti.func_call_with_check')
         return node
+
+    @staticmethod
+    def get_taichi_decorator(node):
```

Comment:
Let's simply reuse this here:
 https://github.com/taichi-dev/taichi/blob/64a612225d08a43d6ce907f196438e70f28c4679/python/taichi/lang/transformer.py#L347